lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -25,6 +26,8 @@
 )
 from .onready import OnDataReady
 
+SID_DIGITS = 2 # number of least significant digits from IMSI to use as table buckets
+N_SHARDS = 10 ** SID_DIGITS # number of shards to distribute UEs
```

Comment:
I suggest moving these to yml config file. Or at least the SID_DIGITS.

Reply:
I am moving SID_DIGITS into yml config. N_SHARDS is dependent on it, I will just create object fields to store them (and set to a default value in the init call if absent).